a3.c

```diff
@@ -135,11 +135,11 @@ void marian_a3_create_controls(struct marian_card* marian) {
 //
 
 
-void marian_a3_prepare(struct marian_card* marian) 
+void marian_a3_prepare(struct marian_card* marian)
 {
 	uint32_t mask = 0x00FFFFFF;
 
-	snd_printdd(KERN_INFO "marian_a3_prepare()\n");
+	MDEBUG("marian_a3_prepare()\n");
```

Comment:
Please use the standard kernel logging and debugging functions instead of custom ones.  If every driver in the kernel had their own way of logging, it would be insane :)